### PR TITLE
Added Swags section & updates socials section

### PR DESCRIPTION
**Pull request by BhaswatiRoy**
Fix: https://github.com/pymc-devs/pymc.io/issues/29

Added swags section & updated socials section with twitter, LinkedIn, youtube, mastodon, meetup links

### CODE REVIEW

**Comment by OriolAbril** [comment]
@BhaswatiRoy do you know how to rebase this branch on the upstream main branch? I just fixed the website build so we need to do this step to see the preview and then be able to merge the PR

**Comment by BhaswatiRoy** [comment]
Hello @OriolAbril 
I tried to update the branch and submit the pr, did it work?

**Comment by OriolAbril** [comment]
Yes, that is great, thanks! You can now see how the page will look like after we merge at: https://pymcio--72.org.readthedocs.build/community/index.html

**Review by OriolAbril — COMMENTED** [review]
I have left two minor comments, but I defer to @reshamas. "swags" sounds very strange to me, but it might be an issue with me having English as 3rd language. Feel free to ignore one or both of them.

File: community/index.md [hunk 1]
```diff
@@ -16,9 +16,21 @@ The PyMC community organizes [PyMCon](https://pymcon.com/), whose last edition w
 PyMC talks have been given at a number of conferences, including [PyCon](https://us.pycon.org/),
 [PyData](https://pydata.org/events/), and [ODSC](https://odsc.com/) events.
 
-## Twitter
+## Swags
```
[the remark this answers is not in the document]
Comment (BhaswatiRoy) [reply community/index.md:19]: Yes we can put "PyMC Merchandise" as heading. Swags seems a bit informal ig

[the remark this answers is not in the document]
Comment (BhaswatiRoy) [reply community/index.md]: Made the required changes

**Pull request merged**